### PR TITLE
MS. Lat. misc. e. 2: Peter Kidd description

### PR DESCRIPTION
**Pull request by holfordm**
Awaiting comment from Nigel Palmer on language of the German text and localization of the main scribe

### CODE REVIEW

File: collections/Lat_misc/MS_Lat_misc_e_2.xml [hunk 1]
```diff
@@ -37,51 +40,215 @@
...
+                     <explicit>qualitatis. quantitatis. temporis. et ordinis. atque boni malive discrecio</explicit>
+                     <finalRubric>Liber Iohannicii explicit</finalRubric>
+                     <bibl>ed. G. Maurach, ‘Johannicius: Isagoge ad Techne Galieni’, <title>Sudhoffs Archiv</title>, 62 (1978), 148–74.</bibl>
                      <textLang mainLang="la">Latin</textLang>
                   </msItem>
                   <msItem n="2" xml:id="MS_Lat_misc_e_2-item2">
-                     <title key="work_12426">Isagoge ad Artem paruum Galeni</title>
-                     <note>tr. <persName key="person_305230031" role="trl"> Constantinus Africanus </persName>
-                     </note>
+                     <locus>(fols. 8v–‘11-12’v)</locus>
```
Comment (adunning) [inline collections/Lat_misc/MS_Lat_misc_e_2.xml:58]: Do we need to adjust the foliation?

File: collections/Lat_misc/MS_Lat_misc_e_2.xml [hunk 2]
```diff
@@ -37,51 +40,215 @@
...
+                     <explicit>defectionem et mortem significat</explicit>
+                     <bibl>Cf. P. Morpurgo, ‘Il commento al de pulsibus Philareti di Mauro Salernitano: Introduzione ed edizione critica dal ms. Parisinus Latinus 18499’, <title>Dynamis: Acta Hispanica ad Medicinae Scientiarumque Historiam Illustrandam</title>, 7–8 (1987–88), 307–46, from 317; using the present MS (siglum ‘L’); among the six MSS. used for comparison with the Paris MS., four others are also at the Bodleian: MS. Ashmole 399, fols. 62r–63r; MS. Ashmole 1285, fols. 181r–182v; MS. Auct. F. 5. 30, fols. 27v–29v; and MS. Laud Lat. 65, fols. 61r–64v.</bibl>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="8" xml:id="MS_Lat_misc_e_2-item8">
+                     <locus>(fols. 35r–54r) </locus>
+                     <author key="person_44299175">Galen</author>
+                     <title key="work_1621">Tegni</title>
+                     <rubric><add place="margin">Tengni G. phylosofi </add></rubric>
+                     <rubric><damage>    </damage> phylosofi</rubric>
```
Comment (adunning) [inline collections/Lat_misc/MS_Lat_misc_e_2.xml:141]: Is the number of spaces meant to indicate the number of missing characters? If so, would `<gap reason="illegible" quantity="3" unit="char"/>` (or `reason="lost"`) be better?

File: collections/Lat_misc/MS_Lat_misc_e_2.xml (hunk 1 above)
Comment (holfordm) [reply collections/Lat_misc/MS_Lat_misc_e_2.xml:58]: We'd have to refoliate the whole volume, so I don't think that would be practical - the '11-12' business is an established Bodleian convention, although many people do dislike / not understand it

File: collections/Lat_misc/MS_Lat_misc_e_2.xml (hunk 2 above)
Comment (holfordm) [reply collections/Lat_misc/MS_Lat_misc_e_2.xml:141]: It was supposed to represent an unspecified amount of damage, but a better solution would just be to have a defective attribute on `rubric`, so I'll change to that.

**Pull request merged**